torch/csrc/jit/passes/quantization.cpp

```diff
@@ -986,10 +988,31 @@ void InsertObserversHelper::preprocess(
   }
 }
 
-// Returns true if the value is the weight to LSTM operator.
-bool isDynamicLSTMWeight(Value* v, Use use, bool is_dynamic) {
-  return is_dynamic && use.user->kind() == Symbol::aten("lstm") &&
-      (use.offset == 2);
+bool useQuantizable(const Use& use, bool is_dynamic) {
+  // Special checks for ops that do not require observers for all input tensors.
+  // For each operator in this list observers are inserted for the input based
+  // on the index specified.
+  const AtenFuncArgs& aten_func_args = AtenFuncArgs({});
+  const CallFuncArgs& call_func_args = CallFuncArgs({});
```

Comment:
could you expose these two lists in the beginning of the file so that it's easier to find and change?

Reply:
Also, a follow up here is, can we remove the isBias check for conv and linear and put the special case in the list?

Reply:
> Also, a follow up here is, can we remove the isBias check for conv and linear and put the special case in the list?

I think we can do this separately, I also see `isWeight` check that uses these lists.